ibm/mas_devops/roles/kmodels/templates/routes/controller.yml.j2

```diff
@@ -0,0 +1,16 @@
+---
+kind: Route
+apiVersion: route.openshift.io/v1
+metadata:
+  name: km-controller
+  namespace: mas-{{ mas_instance_id }}-aibroker
+spec:
+  to:
+    kind: Service
+    name: km-controller
+    weight: 100
+  port:
+    targetPort: http
+  tls:
```

Comment:
this needs to re-encrypt the connection to the backend

Reply:
We are waiting for research team to confirm if there is no issues for set security context for watcher - we will create work item to fix this in future

Reply:
Hi @whitfiea we don't create routes right now